source/h1_connection.c

```diff
@@ -13,6 +13,8 @@
 #include <aws/http/status_code.h>
 #include <aws/io/logging.h>
 
+#include <inttypes.h>
```

Comment:
What made this necessary, it should be covered by common.h?

Reply:
I used `PRIu64` in a LOGF, which comes from `inttypes.h`

you're thinking of stuff like `uint64_t` which comes from `stdint.h`, which we do cover in common.h